### ISSUE TITLE
domain-name-server incorrect in dhclient.conf : spurious \

### ISSUE THREAD

**Issue by ghost** [jdauphant/ansible-role-dns#9]
It is inserted as
<pre>
changed: [bind-host] => (item={u'name': u'domain-name', u'value': u'\\"securedrop.club\\"'})
</pre>
and ends up being
<pre>
supersede domain-name \"securedrop.club\";
</pre>
This is verified with ansible 2.2.3 and ansible 2.4.0. In addition the YAML spec says [there is no need to backslash a double quote within single quotes](http://www.yaml.org/spec/1.2/spec.html#style/flow/single-quoted).

**Comment by jdauphant** [comment]
Thanks for the report @dachary. Have you a solution to suggest as a pull request ?

**Comment by ghost** [comment]
Not really. I ended up using a fork of your project that fixed this, lazy me ;-)

**Comment by jdauphant** [comment]
Laziest persons are better to automate there jobs ;-)
